src/app/qml/filemanager.qml

```diff
@@ -111,7 +111,8 @@ MainView {
         var fileNames = []
         for(var i=0; i < activeTransfer.items.length; i++) {
             var item = activeTransfer.items[i]
-            console.log("Move file to:" + destDir + " with name: " + FmUtils.basename(String(item.url)))
+            auto destFilename = FmUtils.basename(String(item.url));
+            console.log("Move file to:" + destDir + " with name: " + destFilename)
             activeTransfer.items[i].move(destDir, FmUtils.basename(String(item.url)))
```

Comment:
I dont get the point here, you create destFilename and then dont use it in the call to active transfer?

Reply:
Wait, what? Looks a bit fancy, but if you have a look into the overall changes, destFilename is used in activeTransfer.